### PR TITLE
rimage: add src support

### PR DESCRIPTION
**Pull request by gongjun-song**
Add src's params to rimage

Signed-off-by: Gongjun Song <gongjun.song@intel.com>

### CODE REVIEW

**Review by lgirdwood — APPROVED** [review]
@singalsu pls review.

File: config/tgl-cavs.toml [hunk 1]
```diff
@@ -227,3 +227,42 @@ count = 8
...
+		affinity_mask = "0xF"
+		instance_count = "10"
+		domain_types = "1"
+		load_type = "0"
+		module_type = "0x7"
+		auto_start = "0"
+		sched_caps = [1, 0x00008000]
+
+	# pin = [dir, type, sample rate, size, container, channel-cfg]
+	pin = [0, 0, 0xffff, 0xc, 0x8, 0x45ff,
```
Comment (singalsu) [inline config/tgl-cavs.toml:242]: Why are these numbers hex? Hex is OK for bit masks and I/O addresses but not nice for numbers like sample rate.

File: config/tgl-cavs.toml [hunk 2]
```diff
@@ -227,3 +227,42 @@ count = 8
...
+		1, 0, 0xf6c9, 0xc, 0x8, 0x45ff]
+
+	# mod_cfg [PAR_0 PAR_1 PAR_2 PAR_3 IS_BYTES CPS IBS OBS MOD_FLAGS CPC OBLS]
+	mod_cfg = [0, 0, 0, 0, 12832, 1365500, 0, 0, 0, 1365, 0,
```
Comment (singalsu) [inline config/tgl-cavs.toml:246]: Can the numbers be documented better? I have no idea what are the PAR_X and other acronyms.

Comment (RanderWang) [reply config/tgl-cavs.toml:246]: No document. It is copied from module spec and used for fw scheduler by windows. It is not supported by cSOF. use it for windows

File: config/tgl-cavs.toml (hunk 1 above)
Comment (gongjun-song) [reply config/tgl-cavs.toml:242]: This is used for bit masks. Each bit mask represents whether a certain rate is used. For example, suppose the first bit is to judge whether 16khz is used. Setting 1 means using it and setting 0 means not using it.

**Review by singalsu — APPROVED** [review]

**Pull request merged**